### PR TITLE
Plat 606 jempi santempi refactor

### PR DESCRIPTION
**Pull request by nour-borgi**

### CODE REVIEW

**Review by IlyesBenAmara — APPROVED** [review]
LGTM

File: client-registry-santempi/swarm.sh [hunk 1]
```diff
@@ -1,94 +1,122 @@
...
+      "docker service scale instant_$service_name=0" \
+      catch \
+      "Failed to scale down $service_name"
+  done
+}
+
+function destroy_package() {
+  for service_name in "${service_names[@]}"; do
+    docker::service_destroy "$service_name"
+  done
```
Comment (marrouchi) [inline client-registry-santempi/swarm.sh:95]: Should we refactor this into a seperate function that could be used across all packages ? It seems generic enough.

**Review by marrouchi — CHANGES_REQUESTED** [review]

File: client-registry-santempi/swarm.sh (hunk 1 above)
Comment (nour-borgi) [reply client-registry-santempi/swarm.sh:95]: done in another PR :)

**Review by marrouchi — APPROVED** [review]

**Review by Mark-Labuschagne — APPROVED** [review]
Very nice!

**Pull request merged**